model/src/main/java/org/jboss/pnc/model/RepositoryConfiguration.java

```diff
@@ -61,6 +79,15 @@
     private String internalUrl;
 
     /**
+     * Normalized version of scm url to query against. Normalized version is without the protocol and .git extension.
+     */
+    @Size(max = 255)
+    @Column(unique = true, updatable = false)
+    @Getter
+    @Setter
+    private String internalUrlNormalized;
```

Comment:
The original one is required for operations, without it we don't know what the protocol is and what the extension is.